lib/normalizers/v1.js

```diff
@@ -17,6 +17,17 @@ var _ = require('lodash').noConflict(),
         };
     },
 
+    authIdMap = {
+        awsSigV4: 'awsSigV4',
+        basic: 'basicAuth',
+        bearer: 'bearerAuth',
+        digest: 'digestAuth',
+        hawk: 'hawkAuth',
+        ntlm: 'ntlmAuth',
+        oAuth1: 'oAuth1',
```

Comment:
This should be `oauth1` and `oauth2`

Reply:
As discussed, this is the mapping from `helperAttributes.id` to `currentHelper` for v1 normalization.